kong/plugins/datadog/handler.lua

```diff
@@ -53,6 +53,11 @@ local function log(premature, conf, message)
   if premature then
     return
   end
+
+  if message.api == nil then -- unmatched apis are nil
+    return
```

Comment:
I was originally thinking to have a warn log here, however, that would probably be expected and spammy for many.